codes/quantum/oscillators/Homological CV.yml

```diff
@@ -0,0 +1,38 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+code_id: homological_cv
+physical: oscillators
+logical: oscillators
+
+name: 'Homological CV Code'
+
+introduced: '\cite{arXiv:1601.02544}'
+
+description: 'The Homological CV Code construction defines a stabilizer code over \(n\) oscillators to encode a single oscillator state based on some specific chain complex and associated homology. The stabilizer generators are defined by two orthogonal subspaces of the \(C_1\) in the chain complex. \(C_X = \partial_2 C_2\) and \(C_P = \partial_1^T Q\) for some \(Q \subset C_0\). By choosing \(Q \neq C_0\), (unlike the usual approach), we have an extra degree of freedom and get a code.'
```

Comment:
Could you elaborate on the chain complex please? What kinds of operators are the partials? E.g., in homological CSS construction (Sajjad's commit), they are binary matrices.